cli/src/program_v4.rs

```diff
@@ -1107,11 +1117,7 @@ fn send_messages(
             .send_and_confirm_transaction_with_spinner_and_config(
                 &final_tx,
                 config.commitment,
-                RpcSendTransactionConfig {
-                    skip_preflight: true,
-                    preflight_commitment: Some(config.commitment.commitment),
-                    ..RpcSendTransactionConfig::default()
-                },
+                config.rpc_send_transaction_config,
```

Comment:
This is a change in behavior since it used to always skip preflight, but since it's only 1 or 2 transactions, I think it's fine to respect the user's preflight choice. It won't slow things down too much more

Reply:
Oh you mean if someone is calling it with skip_preflight true and before it was ignored and now its used? Should i change something about it?

Reply:
Nah, I think it's fine